hawc/apps/assessment/serializers.py

```diff
@@ -22,6 +22,12 @@ class Meta:
         fields = "__all__"
 
 
+class PublicAssessmentSerializer(serializers.ModelSerializer):
```

Comment:
let's use the existing `AssessmentSerializer` serializer instead of creating a new one